Core/HLE/sceDisplay.cpp

```diff
@@ -1126,8 +1130,12 @@ void Register_sceDisplay_driver() {
 	RegisterModule("sceDisplay_driver", ARRAY_SIZE(sceDisplay), sceDisplay);
 }
 
-void __DisplaySetFramerate(int value) {
-	framerate = value;
+static void __DisplaySetFramerate(void) {
+	if (System_GetPropertyInt(SYSPROP_DEVICE_TYPE) == DEVICE_TYPE_VR)
+		framerate = g_Config.bForce72Hz ? 72 : 60;
+	else
+		framerate = g_Config.iDisplayRefreshRate;
+
 	timePerVblank = 1.001 / (double)framerate;
 	frameMs = 1001.0 / (double)framerate;
 }
```

Comment:
Technically I think there's not much point to applying the "1.001" adjustment at framerates that are different from the original one, since they'll be completely off anyway...

Reply:
Default value is still 60, is it worth it to disable this for other values?

Reply:
If i change these to `1.0`/`1000.0`, stuff runs 0.1% faster according to the debug overlay:

![1 0](https://github.com/user-attachments/assets/dba49075-f1b2-454d-a988-92517ba2353a)

As it currently is, things sit still at 100%:

![1 001](https://github.com/user-attachments/assets/7c43a453-25be-454e-8549-577913c881fb)